virt/virt-4-8-release-notes.adoc

```diff
@@ -145,6 +145,47 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-8-known-issues"]
 == Known issues
 
+* If you initially deployed {VirtProductName} at version 2.4.z or earlier, upgrading to version 4.8 fails with the following message:
++
+[source,terminal]
+----
+risk of data loss updating hyperconvergeds.hco.kubevirt.io: new CRD removes version v1alpha1 that is listed as a stored version on the existing CRD
+----
++
+This bug does not affect clusters where {VirtProductName} was initially deployed at version 2.5.0 or later. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986989[*BZ#1986989*])
++
+** As a workaround, remove the `v1alpha1` version and resume the upgrade process.
++
+. Run a proxy to the Kubernetes API server by running the following command:
```

Comment:
AFAIK, this command is for opening a proxy session with the cluster, not for communicating directly with the Kubernetes API server. Suggest "Open a proxy connection to the cluster" or "Start a proxy session with the cluster" or something similar.

Reply:
oc help says:
```
$ oc help | grep proxy
  proxy           Run a proxy to the Kubernetes API server
```

Reply:
Help and man files are usually written in a very concise style. See https://www.redhat.com/sysadmin/openshift-terminating-state and https://cloud.redhat.com/blog/full-cluster-capacity-management-monitoring-openshift. 

RE: "run". You can run a proxy server (intransitive) but running a proxy TO a server is not normal English usage. 

The oc developer commands (oc-by-example-content.adoc) do not follow our documentation conventions, so I do not recommend following their wording.

== oc proxy
Run a proxy to the Kubernetes API server

.Example usage
[source,bash,options="nowrap"]
----
  # To proxy all of the kubernetes api and nothing else.
  oc proxy --api-prefix=/
  
  # To proxy only part of the kubernetes api and also some static files.
  # You can get pods info with 'curl localhost:8001/api/v1/pods'
  oc proxy --www=/my/files --www-prefix=/static/ --api-prefix=/api/
  
  # To proxy the entire kubernetes api at a different root.
  # You can get pods info with 'curl localhost:8001/custom/api/v1/pods'
  oc proxy --api-prefix=/custom/

Reply:
Updating this to "Open a proxy connection to the cluster"